app/services/one_off_redirect_fix_service.rb

```diff
@@ -0,0 +1,43 @@
+class OneOffRedirectFixService
+  def self.fix_redirects!
+    editions = Edition.where(document_type: "redirect").joins(:document).where.not("documents.locale = 'en'")
+    editions.each do |edition|
+      updated = update_redirects(edition)
+      Commands::V2::RepresentDownstream.new.call([edition.content_id], queue: DownstreamQueue::LOW_QUEUE) if updated
```

Comment:
Might be nice to keep track of a `editions_updated += 1` variable here, and output it at the end of the task, so we know how many have been updated 👀